config/tests/snapshots/config_tests__committee.snap

```diff
@@ -4,7 +4,7 @@ expression: committee
 ---
 {
   "authorities": {
-    "IP26ybELdYe7p7W8FjvOaeeW1x5O1EwQ/LRIhon3oUQ=": {
+    "ildi4hrBzbOHBELHe0w69Yx87bh3nQJw5tTx4vc2fXQ=": {
```

Comment:
this is expected as rand 0.8.5 uses ChaChaRng for 20 rounds whereas rand 0.7.3 uses 12 rounds with reference to https://github.com/rust-random/rand/issues/932

from 0.7.3: 
```
/// The current algorithm used is the ChaCha block cipher with 20 rounds.
/// This may change as new evidence of cipher security and performance
/// becomes available.
```

from 0.8.5:
```
/// The current algorithm used is the ChaCha block cipher with 12 rounds. Please
/// see this relevant [rand issue] for the discussion. This may change as new 
/// evidence of cipher security and performance becomes available.

/// [rand issue]: https://github.com/rust-random/rand/issues/932
```

Reply:
yeah, only for the generate function in KeyPair trait that takes in the Rng